utilities/transactions/optimistic_transaction.cc

```diff
@@ -60,12 +60,13 @@ Status OptimisticTransaction::Commit() {
                                             OptimisticTransactionDB>(txn_db_);
   assert(txn_db_impl);
   OccValidationPolicy policy = txn_db_impl->GetValidatePolicy();
-  if (policy == OccValidationPolicy::kValidateParallel) {
-    return CommitWithParallelValidate();
-  } else if (policy == OccValidationPolicy::kValidateSerial) {
-    return CommitWithSerialValidate();
-  } else {
-    assert(0);
+  switch (policy) {
```

Comment:
Minor: you can move `txn_db_impl->GetValidatePolicy()` to here too.

Reply:
done